### DISCUSSION TITLE
Información de como aser solicitud para poder tomar mis ganancias de Bitcoin cómo poder aseder. a cuenta paynet

### DISCUSSION THREAD

**Discussion by ID81214283** [btcpayserver/btcpayserver#5040] [Technical Support] [answered]
Para poder tomar mis ganancias y retirarlos ,en una cuenta de pagó paynet como yenar formulario para poder cobrar pagos

**Comment by Kukks** [comment, marked as answer]
https://blog.btcpayserver.org/bitcoin-bonus-scam
